tests/Base64.test.ts

```diff
@@ -0,0 +1,96 @@
+import * as RA from 'fp-ts/ReadonlyArray'
```

Comment:
🔧 I think it might be good to add more tests for `{ urlSafe: false }`.  This might be more natural to test if we break up the module into two different modules

https://github.com/validatorjs/validator.js/blob/86a07ba4f3f710f639e92a62cf81dd3321ef9ee8/test/validators.js#L5955